tekton/src/tasks/gitops/gitops-db2u-database.yml.j2

```diff
@@ -36,6 +36,18 @@ spec:
       type: string
     - name: db2_backup_storage_class
       type: string
+    - name: db2_meta_storage_size
+      type: string
+    - name: db2_temp_storage_size
+      type: string
+    - name: db2_logs_storage_size
+      type: string
+    - name: db2_audit_logs_storage_size
+      type: string
+    - name: db2_data_storage_size
+      type: string
+    - name: db2_backup_storage_size
+      type: string
```

Comment:
@whitfiea should we add defaults on to these as well?

Reply:
those should inherit the default from the pipeline, so it isn't needed unless the task is used in a different pipeline